api.c

```diff
@@ -521,6 +521,14 @@ int tcmu_emulate_evpd_inquiry(
 		/* Optimal xfer length */
 		memcpy(&data[12], &val32, 4);
 
+		/* MAXIMUM UNMAP LBA COUNT */
+		val32 = htobe32(VPD_MAX_UNMAP_LBA_COUNT);
+		memcpy(&data[20], &val32, 4);
+
+		/* MAXIMUM UNMAP BLOCK DESCRIPTOR COUNT */
+		val32 = htobe32(VPD_MAX_UNMAP_BLOCK_DESC_COUNT);
+		memcpy(&data[24], &val32, 4);
+
 		tcmu_memcpy_into_iovec(iovec, iov_cnt, data, sizeof(data));
 
 		return SAM_STAT_GOOD;
```

Comment:
Yeah, it is.

And together with the LBPU fileld will set later in another patch.